setup.cfg

```diff
@@ -34,6 +34,7 @@ testall =
     asdf
     gwcs
     pyvo
+    matplotlib
```

Comment:
Is this because of the sunpy tests? If so I fixed it in #250 by adding the `[map]` extra.

Reply:
Yes I'll remove this, thanks!